pkgs/top-level/all-packages.nix

```diff
@@ -39142,7 +39144,10 @@ with pkgs;
 
   stayrtr = callPackage ../servers/stayrtr {};
 
-  sunshine = callPackage ../servers/sunshine {};
+  sunshine = callPackage ../servers/sunshine {
+    ffmpeg_5-full = ffmpeg_5-full.override { nv-codec-headers = nv-codec-headers-11; };
+    libcbs = callPackage ../servers/sunshine/libcbs.nix { };
```

Comment:
We should make this accessible to all packages or move it into the sunshine file.

Reply:
Moved into the main file.